### ISSUE TITLE
Obeytabs loses multiline comments started with tabs

### ISSUE THREAD

**Issue by LDVSOFT** [gpoore/minted#88]
Try to typeset:
````tex
\documentclass{article}
\usepackage{minted}
\begin{document}
\begin{minted}[obeytabs]{java}
// Test 1
	// Test 2 (tab in the beggining)
    // Test 3 (spaces)
/*
Test 4
*/
	/*
	Test 5 (tabs here too!)
	*/
    /*
    Test 6 (spaces here)
    */
class Main {}
\end{minted}
\end{document}
````
You'll be a little disappointed: all Test 5, except first `/*`, will be replaced with empty lines.
Without `obeytabs` it's fine.

`pygments` version 2.0.2, `minted` version 2.0.
`-8bit` enabled.

**Comment by gpoore** [comment]
This is due to an issue with `fancyvrb`, which `minted` uses internally to typeset code. `fancyvrb` has been essentially unchanged since 1998, so any fixes for it are unlikely. A patch for `fancyvrb` on the `minted` side is theoretically possible, but would probably be quite complicated and is unlikely in the near future since a totally new algorithm would probably be needed. If you can give me a little more detail about why you need `obeytabs`, I may be able to suggest alternatives.

This should be added to the documentation as a limitation on `obeytabs`.

**Technical details for future reference**: `obeytabs` is done by `fancyvrb`, which `minted` uses internally. The algorithm for tab expansion seems to assume that tabs are not inside a command. The algorithm also assumes that the macro `\FancyVerbFormatLine` is not overwritten by the user. (The relevant part of the [`fancyvrb` documentation](https://www.ctan.org/pkg/fancyvrb) is pages 43-44.) However, Pygments is returning the leading tab *inside* a command, for example (GitHub is converting the tab into a space):
```latex
\PYG{c+cm}{	Test 5  (tabs here too!)}
```
This breaks the `fancyvrb` code. If Pygments put the tab outside the command, then everything would probably work. But since the tab is actually inside the comment, what Pygments is doing makes sense, even if it would be possible to make a case for doing things differently.

**Comment by LDVSOFT** [comment]
Hm. Possible hotfix could be a regexp moving tabs outside of commands, but hey.

For now, I will try to switch off `obeytabs` in my documents.

**Comment by gpoore** [comment]
I've added a note about this to the docs. I'm closing this issue. Further discussion about `obeytabs` will be in #99, which gets into even broader problems with the `obeytabs` option.

**Comment by gpoore** [comment]
This is fixed in [v2.3](https://github.com/gpoore/minted/releases/tag/v2.3). Tabs will no longer cause lines to vanish. Tabs that are used for indentation will work with `obeytabs` and use correct tab stops. Tabs that follow anything besides spaces or tabs (that is, not part of indentation) won't necessarily use correct tab stops.
